fifteen_min/src/viewer.rs

```diff
@@ -344,7 +344,7 @@ fn build_panel(ctx: &mut EventCtx, app: &App, start: &Building, isochrone: &Isoc
     }
 
     // Start of toolbar
-    rows.push(Widget::horiz_separator(ctx, 0.3).margin_above(10));
```

Comment:
potentially controversial: I made the size of the horizontal separator relative to its parent size, rather than relative to the window size.

Reply:
That makes total sense. I think most of the early uses happened to be in containers that spanned most of the window, so the difference didn't matter much